src/components/picker/index.tsx

```diff
@@ -190,6 +197,8 @@ const Picker = (props: PropsWithChildren<PickerProps> & ForwardRefInjectedProps
     return (
       <PickerItemsList
         testID={`${testID}.modal`}
+        useWheelPicker={useWheelPicker}
+        items={items}
```

Comment:
Shouldn't you pass 'items' only if 'useWheelPicker' is true?

Reply:
I can add a condition, but it doesn't really matter or affect the code.

Reply:
I think it's better to secure no side-effects